src/http/AuthHeader.js

```diff
@@ -26,36 +26,35 @@ class AuthHeader {
      * @param {AuthContext} context - auth context for access token redemption
      */
     async addAuthorizationHeader(memberId, request, context) {
-        let now = new Date().getTime();
+        const now = new Date().getTime();
 
         // Parses out the base uri
         let uriPath = request.url.replace(this._baseUrl, '');
-
         // Makes sure the uri is formatted correctly
-        uriPath = uriPath.substring(0, 1) === '/' ? uriPath : uriPath + '/';
+        uriPath = uriPath.substring(0, 1) === '/' ? uriPath : `/${uriPath}`;
```

Comment:
omg, was this wrong before?

Reply:
yeah its really weird, took me a long time to catch this, not sure why it wasnt failing before